spec/support/feature_flags_helper.rb

```diff
@@ -0,0 +1,10 @@
+module FeatureFlagsHelper
+  def set_feature_flag(user, feature, status)
+    feature_flags = instance_double(FeatureFlags, "#{feature}?" => status)
+    allow(FeatureFlags).to receive(:new).with(user) { feature_flags }
+  end
+
+  def unstub_feature_flags(user)
+    allow(FeatureFlags).to receive(:new).with(user).and_call_original
+  end
+end
```

Comment:
I'm not sure it's really worth introducing new methods for what `allow` already does and with which all devs are familiar with. If feature flags were more tedious it'd make more sense.

Reply:
After thinking it over, I'm okay with removing these spec helpers. Done! Thanks, @sauloperez.